defaults/main.yml

```diff
@@ -33,3 +33,17 @@ riemann_slack_tag: slack_alert
 
 # Disable verbose logging, useful for standing up the server.
 riemann_debug_logging: false
+
+riemann_create_folders:
+  - "conf.d"
+  - "utils"
```

Comment:
Why these two separate directories for config items? Do the Riemann docs recommend such a breakdown? Is there any reason we can't just dump everything in `conf.d/` via a fileglob and call it a day?